server/jetstream_api.go

```diff
@@ -3320,6 +3325,13 @@ func (s *Server) jsConsumerInfoRequest(sub *subscription, c *client, subject, re
 				resp.Error = jsClusterNotAvailErr
 				s.sendAPIErrResponse(ci, acc, subject, reply, string(msg), s.jsonResponse(&resp))
 			}
```

Comment:
Do we need a `return` after sending the error in the `if js.isLeaderLess()` case?

Reply:
Will reflow this part.